### PR TITLE
List

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review

### CODE REVIEW

File: List.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    gap: 0.5rem;
+    justify-content: center;
+    margin: .5rem;
+    padding: 1rem;
+    width: 50rem;
+    height: auto;
+  }
+  .list-title {
+    padding: 1rem;
+    font-weight: 5px;
```
Comment (anshulraheja) [inline List.css]: font-weight is not given in pixel

File: List.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    border-bottom: 1px solid black;
+    margin-bottom: 0.5rem;
+  }
+  .list {
+    background-color: var(--component-white-color);
+    padding: 1rem 0;
+    margin: 2.5rem;
+    width: 15.6rem;
+    max-width: 90%;
```
Comment (anshulraheja) [inline List.css]: According to me, the width should be 90% and the max-width should be 15.6rem. Try it out

File: List.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  }
+  .list-style {
+    background-color: var(--component-white-color);
+    padding: 1rem 1rem;
+    padding-left: 0;
```
Comment (anshulraheja) [inline List.css]: left padding of 1rem given on line 30 is getting over write bt line no 31. You can use shorthand for padding.

File: List.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    margin: 2.5rem;
+    width: 20rem;
+    max-width: 100%;
```
Comment (anshulraheja) [inline List.css]: same interchange values of widths. it will be then become a responsive site

File: List.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  }
+  .list li {
+    font-size: 0.8rem;
+    list-style: none;
+    padding: 0.7rem 1.1rem;
+    font-weight: 4px;
```
Comment (anshulraheja) [inline List.css]: font-weight issue

**Review by anshulraheja — APPROVED** [review]

File: List.css [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  }
+  .list-style li {
+    font-size: 0.8rem;
+    padding: 0.7rem;
+    font-weight: 4px;
```
Comment (gadgetfather) [inline List.css]: give correct font weight

File: List.html [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+          </ul>  
+            <div class="nested-list">Watches</div>
+          <ul>  
+          <li>Analog</li>
+          <li>Digital</li>
+          </ul>
+      </div>
+      <div class="wrapper">
+        <!-- Simple List -->
+        <div class="list-container border-style">
```
Comment (gadgetfather) [inline List.html:87]: very informative class names.

**Review by gadgetfather — CHANGES_REQUESTED** [review]

File: List.css (hunk 1 above)
Comment (ShrutiLakhani) [reply List.css]: Done!

**Review by gadgetfather — APPROVED** [review]
PR is good to merge

File: List.css (hunk 2 above)
Comment (ShrutiLakhani) [reply List.css]: Done! Thank you for pointing it out.

File: List.css (hunk 3 above)
Comment (ShrutiLakhani) [reply List.css]: Done

File: List.css (hunk 4 above)
Comment (ShrutiLakhani) [reply List.css]: Done

File: List.css (hunk 5 above)
Comment (ShrutiLakhani) [reply List.css]: Done

File: List.css (hunk 6 above)
Comment (ShrutiLakhani) [reply List.css]: Done

**Pull request merged**